src/main/java/com/work/management/service/employee/EmployeeService.java

```diff
@@ -0,0 +1,7 @@
+package com.work.management.service.employee;
+
+import com.work.management.dto.EmployeeDto;
+
+public interface EmployeeService<EmployeeDto, Employee> {
+    void save(com.work.management.dto.EmployeeDto document);
```

Comment:
same here..as below... and change com.work.management.EmpolyeeDto to just EmployeeDto